### DISCUSSION TITLE
Add mute button for videos

### DISCUSSION THREAD

**Discussion by CodeShakingSheep** [libre-tube/LibreTube#6497] [Ideas]
### Describe your suggested feature

Sometimes I want to watch a video without the audio while listening to music with another app. This is currently not possible with LibreTube as I can only turn off the volume for my whole phone (in this case I cannot hear the music either). This scenario works with Clipious as Clipious has a mute button next to its fullscreen button in videos. So, my suggestion for LibreTube is to add a mute button as well.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
That's likely useless for the average user and thus not planned for the moment.
